src/ProjectService.Models.Db/DbTask.cs

```diff
@@ -33,6 +34,14 @@ public class DbTask
         public DbTask ParentTask { get; set; }
 
         public ICollection<DbTask> Subtasks { get; set; }
+
+        [IgnoreParse]
```

Comment:
do you know for what this attribute?

Reply:
As far as I understand, we use this attribute for dependent entities.